### DISCUSSION TITLE
Allow Changing Locale By Passing In Locale String

### DISCUSSION THREAD

**Discussion by Rishi556** [KevinVandy/material-react-table#1174] [Ideas]
Rather than importing and passing the entire localization values, a developer should be able to simply pass in the locale string to make translation simpler.

Example usage:

```js
const table = useMaterialReactTable({
  columns,
  data,
  localization: "es",
});
```

**Comment by KevinVandy** [comment]
No, because that would double the bundle size by including all locales internally.
